src/terragraph-e2e/e2e/if/Controller.thrift

```diff
@@ -697,6 +697,7 @@ enum CfgAction {
   RELOAD_TUNNEL_CONFIG = 130,
   RELOAD_VPP_CONFIG_AND_MONITOR = 140,
   UPDATE_ZONE = 141,
+  RELOAD_TOPOLOGY_NAMES = 150,
```

Comment:
- You also need to update docs in `docs/developer/Configuration_Management.md`
- Please find a more suitable place, ex. any of `33`, `74`, `142`

Reply:
Acknowledged, working on review comments.

Reply:
Addressed the comments